htdocs/core/lib/functions.lib.php

```diff
@@ -12154,7 +12166,15 @@ function getElementProperties($element_type)
 		'dir_output' => $dir_output
 	);
 
-	//var_dump($element_properties);
+	$reshook = $hookmanager->executeHooks('changeElementProperties', $parameters);
```

Comment:
Can youbrename trigger function in getElementProperties to match same name than dolibarr function.